server-nodejs/extensions/jwt.js

```diff
@@ -60,15 +60,10 @@ module.exports = function (app) {
                 user.lastlogin = Date.now();
                 db.update([user], function(err, nodes) {});
                 debug('User authenticated, generating token');
-                user.token = jwt.sign(payload, conf.secret + user.password, options);
-                var decoded = jwt.decode(user.token);
-                user.token_exp = decoded.exp;
-                user.token_iat = decoded.iat;
-                delete user.password;
+                user.address = req.connection.remoteAddress;
+                user.class = req.user.class || 'guest';
```

Comment:
is req.user.class defined?

Reply:
Yes is not defined. Without "req", is defined.